esp/smc/SMCLib/TpWrapper.hpp

```diff
@@ -211,6 +211,19 @@ class CCluster
 };
 
 extern TPWRAPPER_API ISashaCommand* archiveOrRestoreWorkunits(StringArray& wuids, IProperties* params, bool archive, bool dfu);
+
+interface IConstContainerWUClusterInfo : extends IInterface
+{
+    virtual IStringVal& getName(IStringVal& str) const = 0;
+    virtual IStringVal& getThorQueue(IStringVal& str) const = 0;
+    virtual IStringVal& getAgentQueue(IStringVal& str) const = 0;
+    virtual IStringVal& getServerQueue(IStringVal& str) const = 0;
+    virtual ClusterType getPlatform() const = 0;
+};
+typedef IArrayOf<IConstContainerWUClusterInfo> CConstContainerWUClusterInfoArray;
```

Comment:
would be okay to use existing CConstWUClusterInfoArray (IArrayOf<IConstWUClusterInfo>) for this stop-gap implementation too.